.github/workflows/iroha2-dev-pr.yml

```diff
@@ -95,6 +95,61 @@ jobs:
           # This context specification is required
           context: .
 
+  workspace_analysis_clippy:
```

Comment:
Is there any reason to insert these jobs in this position?
Since jobs run in parallel, their order wouldn't affect functionality, but from a documentation standpoint, the order used to be `consistency`, `with_coverage`, and `integration` before #4605.
It looks to me like a sequence from static/simple to dynamic/complex tests

Reply:
moved it up